### ISSUE TITLE
How to add logo inside

### ISSUE THREAD

**Issue by stefandevo** [guitarrapc/SkiaSharp.QrCode#7]
The referenced https://github.com/codebude/QRCoder has option to set a logo inside of the QR.
Is it supported with this lib?

**Comment by guitarrapc** [comment]
It isnt support on this lib. I knew the spec but there are no requirement on my use case.

**Comment by codebaard** [comment]
When I create a PNG-Output, I'll get a PNG with alpha = 0 (transparent). Is there a way to set alpha = 1 (fully opaque)?

Also: If just use the option JPEG Output, I get a fully black Image. Is this an actual Issue or supposed to be that way?

Thanks in Advance.

**Comment by AlexGipp** [comment]
@stefandevo @guitarrapc im currently working on an implementation for icons, as soon as im done i will open up a merge request.

**Comment by codebaard** [comment]
> @stefandevo @guitarrapc im currently working on an implementation for icons, as soon as im done i will open up a merge request.

Please keep me in the loop. This might become useful for me soon, too. Your work is really much appreciated.

**Comment by AlexGipp** [comment]
> 
> 
> > @stefandevo @guitarrapc im currently working on an implementation for icons, as soon as im done i will open up a merge request.
> 
> Please keep me in the loop. This might become useful for me soon, too. Your work is really much appreciated.

@codebaard i just opened up a merge request with the changes 👍

**Comment by guitarrapc** [comment]
@stefandevo  @AlexGipp I've released 0.4.0 which include icon supports.
You can add logo with SKCanvas.Render.

```csharp
var logo = File.ReadAllBytes("samples/test.png");
var icon = new IconData
{
    Icon = SKBitmap.Decode(logo),
    IconSizePercent = 10,
};
canvas.Render(qr, info.Width, info.Height, SKColor.Empty, SKColor.Parse("000000"), icon);
```

thanks.
